notebooks/mos_spectroscopy_advanced/MOSspec_advanced.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    if region1_exists==False:
</pre><p> </p><p>I would suggest using "not" instead of "== False" for more concise code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
Done, thank you!